pandas/core/categorical.py

```diff
@@ -1180,7 +1180,9 @@ def fillna(self, fill_value=None, method=None, limit=None):
         value : scalar
             Value to use to fill holes (e.g. 0)
         limit : int, default None
-            Maximum size gap to forward or backward fill (not implemented yet!)
+            If method is specified, this is the maximum size gap to forward or backward
+            fill. Otherwise this is the maximum number of entries along the axis where
+            holes will be filled. (not implemented yet!)
```

Comment:
Maybe put the not implemented yet first? (and maybe also explicitely "for Categorical")